src/browser/api_protocol/api_handlers/system.js

```diff
@@ -41,6 +41,7 @@ function SystemApiHandler() {
         'get-el-ipc-config': getElIPCConfig,
         'get-environment-variable': { apiFunc: getEnvironmentVariable, apiPath: '.getEnvironmentVariable' },
         'get-host-specs': { apiFunc: getHostSpecs, apiPath: '.getHostSpecs' },
+        'get-min-log-level': getMinLogLevel,
```

Comment:
I think min is more natural, what are the chances that you want to set `info` log levels and not get 'error' as well?